tests/test_util_functions.py

```diff
@@ -127,7 +127,7 @@ def test_get_available_idps(self):
     @ddt.data(
         (EnterpriseCustomer, [
             "enterprisecustomeruser", "pendingenterprisecustomeruser", "branding_configuration", "created", "modified",
-            "uuid", "name", "active", "identity_provider", "site"
+            "uuid", "name", "catalog", "active", "identity_provider", "site"
```

Comment:
@haikuginger suggestion: using actual model doesn't occur to me as a good idea anymore - maybe we should switch to synthetic test-only models here, instead of actual ones? Feel free to skip/ignore this comment.